base/complex.jl

```diff
@@ -384,9 +409,7 @@ function /(z::ComplexF64, w::ComplexF64)
         c*=bs;  d*=bs;  s*=bs   # scale up c,d
     end
 
-    # division operations
-    abs(d)<=abs(c) ? ((p,q)=robust_cdiv1(a,b,c,d)  ) : ((p,q)=robust_cdiv1(b,a,d,c); q=-q)
-    return ComplexF64(p*s,q*s) # undo scaling
+    return a,b,c,d,s
 end
 function robust_cdiv1(a::Float64, b::Float64, c::Float64, d::Float64)
```

Comment:
This doesn't seem to get inlined.

Reply:
You could use `@inline` to force the issue.